source/NoteApplication.h

```diff
@@ -40,11 +43,13 @@ class NoteApplication : public BApplication {
 				void		OpenNote(entry_ref *ref);
```

Comment:
Remove the default constructor and:
```suggestion
				void		OpenNote(entry_ref* ref = NULL);
```